File: linear search/Java/LinearSearch.java
class LinearSearch{
  
  public static int search(int array[], int target){

    for(int i = 0; i < array.length; i++){
      if(array[i] == target){
        return i;
      }
    }
    return -1;
  }

  public static void verify(int index){
    if (index != -1){
      System.out.println("Target found at index: " + index);
    }else{
      System.out.println("Target is not found.");
    }
  }
  public static void main(String[] args){
    int number [] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
    int result_1 = search(number, 5);
    verify(result_1);

    int result_2 = search(number, 11);
    verify(result_2);
  }

}

File: Algorithms/linear search/Python/linear_search.py
def linear_search(list, target):
  "Returns the index position of the target if found, else returns None"
  for item in range(0, len(list)):
    if list[item] == target:
      return item
  return None


def verify(index):
  "Verifies if the index exits"
  if index is not None:
    print("Target found at index ", index)
  else:
    print("Target not found.")


numbers = [num for num in range(1, 20)]

result_1 = linear_search(numbers,  12)
result_2 = linear_search(numbers,  25)

verify(result_1)
verify(result_2)